### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by predrag-babic**
add headline and hints to github-and-markdown

### CODE REVIEW

**Comment by Alla-Nazarenko** [comment]
Hallo!

File: github-and-markdown/github-and-markdown.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Github-and markdown
+
+-Working easy with GitHub
+
+## Useful commands
+
+-git add
```
Comment (Leon910) [inline github-and-markdown/github-and-markdown.md]: that looks good but you also need the file-name?

**Review by Leon910 — COMMENTED** [review]
review for line 7

File: github-and-markdown/github-and-markdown.md (hunk 1 above)
Comment (predrag-babic) [reply github-and-markdown/github-and-markdown.md]: Thanks for the hint, i´ve changed the command.

**Pull request merged**